playground/src/index.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+window.React = React;
+/**
+ * External dependencies
+ */
+import '@babel/polyfill';
+
+/**
+ * WordPress dependencies
+ */
+import '@wordpress/editor'; // This shouldn't be necessary
```

Comment:
We're still using `core/editor` in some places in the block editor, it should be refactored properly to avoid the need to import this.